HistoryService/client/components/institution.jsx

```diff
@@ -25,47 +26,36 @@ class Institution extends React.Component {
 
     let myInit = { method: "GET", headers: myHeaders };
     let that = this;
-
-    fetch("/api/articles/" + that.props.params.articleId, myInit)
-      .then(function(response) {
-        return response.json();
-      })
-      .then(function(response) {
-        if (response.error.error) {
-          StatusAlertService.showError(response.error.message);
-        } else {
-          that.setState({ article: response.data });
-        }
-        that.setState({ loading: false });
+    let url =
```

Comment:
If another url variable is written underneath, maybe renaming the variable to make it more clear will help with readability (i.e. `archive_url`)